src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.ul__list {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.list {
+  margin-right: 20px;
+  list-style-type: none;
+}
+
+.link {
+  position: relative;
+  display: block;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  color: #000;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 14px;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 23px 0;
+}
+
+.list:last-child {
+  margin-right: 0;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.link:active::after,
```

Comment:
what's going on here? why do you additionally use `.link:active::after`  explain